File: Android-VS-iOS/Java_and_swift_test_cases/Inheritance/Calculation.swift
import UIKit

class Calculation{
	var z:Int = 0
	public func addition(ab:Int, y:Int) {
		z = ab + y
		print(z)
	}

	public func Subtraction(x:Int, y:Int) {
		z = x - y
		print(z)
	}

}
 class My_Calculation: Calculation{
	public func multiplication(x:Int, yy:Int) {
		z = x * yy
		print(z)
	}

	
}
var a:Int = 20, b:Int = 10
var demo:My_Calculation = My_Calculation()
demo.addition(ab:a ,y:b)
demo.Subtraction(x:a ,y:b)
demo.multiplication(x:a ,yy:b)

File: Android-VS-iOS/Java_and_swift_test_cases/Inheritance/Calculation.java

class Calculation {
    int z = 0;

    public void addition(int ab, int y) {
        z = ab + y;
        System.out.println(z);
    }

    public void Subtraction(int x, int y) {
        z = x - y;
        System.out.println(z);
    }
}

public class My_Calculation extends Calculation {
    public void multiplication(int x, int yy) {
        z = x * yy;
        System.out.println(z);
    }

    public static void main(String args[]) {
        int a = 20, b = 10;
        My_Calculation demo = new My_Calculation();
        demo.addition(a, b);
        demo.Subtraction(a, b);
        demo.multiplication(a, b);
    }
}